.idea/libraries/kxml2.xml

```diff
@@ -4,6 +4,8 @@
       <root url="jar://$PROJECT_DIR$/android/android/lib/kxml2-2.3.0.jar!/" />
```

Comment:
It seems the kxm2 library comes from (and used only in) android subsystem. If we want to use it in another subsystems I would move it to community/lib